meteor-app/imports/ui/Login/login.tsx

```diff
@@ -26,12 +27,20 @@ export const Login = () => {
   }
 
 	const handleSubmit = () => {
-		setUsername('') 
-		setPassword('')
-		Meteor.loginWithPassword(username, password, (error) => {
-			if(error) console.log(error);
-			else history.push('report-list')
-		})
+		
+		if (!username || !password) {
+			toast.error('missing username or password')
+		} else {
+			setUsername('') 
```

Comment:
You should probably move these state setters so they run after loginWithPassword runs without error. 

So reset the state, then route to report-list